### ISSUE TITLE
DatePicker minDate and maxDate Attributes Off

### ISSUE THREAD

**Issue by dbarbour** [NativeScript/docs-v7#1363]
### Please, tell us what's the problem?
[ ] A typo
[X] Wrong documentation
[ ] Improvement of existing article
[ ] Missing documentation
[ ] New article needed

### Please, tell us more details

DatePicker 
year="1975" month="01" day="11" 
minDate="1975-01-11" maxDate="1975-01-16"

This yields a DatePicker with the days restricted to 10, 11, 12, 13, 14, 15.   

I would have expected: 11, 12, 13, 14, 15, 16

In summary, the day value of minDate and maxDate seem to be off by one day.

**Comment by etabakov** [comment]
Hey @dbarbour, I just tried to reproduce the problem you are experiencing in [this Playground](https://play.nativescript.org/?template=play-ng&id=bulzZN), but the DatePicker behaves as expected. Can you send us a repro?

**Comment by dbarbour** [comment]
The Playground is a smart way to communicate.  Will do that in the future!

In the Playground example I get the following days: 10 to 15.   On two Android devices.

I would have expected minDate="1975-01-11" maxDate="1975-01-16" would have the days 11 to 16, right?

**Comment by dbarbour** [comment]
I updated the playground with more demos including the input and the output data for the day.

==more info provided==  ( don't know if/how to remove the 'need more info' label )

**Comment by etabakov** [comment]
Just tested your Playground both with iOS and Android and it still works as expected. Can you provide more information about the device you are using:
 - Platform: iOS/Android
 - Platform version
 - Regional settings and timezone settings 
 - anything else that you think might be relevant to reproducing the issue
 - if you can share a video with the problem would be helpful as well
